docs/output.md

```diff
@@ -210,3 +210,51 @@ Deduplication is carried by two possible tools, as described below. However the
 The resulting histogram file will contain estimated deduplication statistics at different theoretical sequencing depths, and can be used to generate a complexity curve for estimating the amount unique reads that will be yield if the library is re-sequenced.
 
 These curves will be displayed in the pipeline run's MultiQC report, however you can also use this file for plotting yourself for further exploration e.g. in R to find your sequencing target depth.
+
+### Damage Manipulation
+
+There are three different option for manipulation of ancient DNA damage.
+
+#### Damage Rescaling
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `damage_manipulation/`
+
+  - `*_rescaled.bam`: Reads with their base qualities rescaled according to the bayesian aDNA damage model, in BAM format.
+  - `*_rescaled.bam.{bai,csi}`: Index file corresponding to the BAM file.
+
```

Comment:
The damageplots etc. aren't as important, I mean just actual log files that might be useful, e.g.  DamageProfiler emits

```
    tuple val(meta), path("results_*/Runtime_log.txt")                                    ,emit: runtime_log
```

Reply:
There are multiple `Stats_out_MCMC_*` files that provide information on the damage prediction algorithm. They were not published in previous versions of nf-core/eager. Additionally, it would be multiple files per input bam (3 plots and 3 CSVs of plot data), which I think will clutter the output folder a lot.